common/install_azure_persistent_rdma_naming.sh

```diff
@@ -68,3 +69,5 @@ EOF
 
 systemctl enable azure_persistent_rdma_naming.service
 systemctl start azure_persistent_rdma_naming.service
+
+sudo ./install_azure_persistent_rdma_naming_monitor.sh      
```

Comment:
no need for absolute paths starting from $COMMON_DIR/install_azure_persistent_rdma_naming_monitor.sh  ?

Reply:
Ah we should be able to use $COMMON_DIR here I believe, however I was unable to test with the value -> I will change this to use $COMMON_DIR